server/lib/chat/chat-models.ts

```diff
@@ -578,3 +579,94 @@ export async function findChannelByName(
     done()
   }
 }
+
+/** Returns the number of channels that match a particular condition. */
+async function getChannelsCount(
```

Comment:
Weird name, maybe `countMatchingChannels`? I'm not sure that I really like this API at all though, it feels like My First SQL Injection. Would prefer writing targeted (static) queries rather than making queries we just plug random SQL into